client/components/Home/Home.js

```diff
@@ -36,21 +43,6 @@ class Home extends Component {
       filters[event.target.name].splice(index, 1)
       this.setState({ filters })
     }
-  }
-
-  renderList (items, type) {
-    return items.map((item, index) => {
-      return (
-        <FormControlLabel key={index}
-          control={ <Checkbox name={type} value={item} onChange={this.handleCheckbox} /> }
-          label={item}
-          className={styles.checkBoxes}
-        />
-      )
-    })
-  }
-
-  render () {
     const filteredProjects = this.props.projects.filter(project => {
```

Comment:
Still regarding this point.... look at this code and tell me if you agree and like it
```

projectContainsCause (project) {
    return this.state.filters.causes.length === 0 || this.state.filters.causes.some(cause => project.causes.indexOf(cause) >= 0)
  }
  projectContainsTech (project) {
    return this.state.filters.technologies.length === 0 || this.state.filters.technologies.some(tech => project.technologies.indexOf(tech) >= 0)
  }

  handleCheckbox (event, checked) {
    const filters = this.state.filters
    const index = this.state.filters[event.target.name].indexOf(event.target.value)
    if (index === -1) {
      filters[event.target.name].push(event.target.value)
      this.setState({ filters })
    } else {
      filters[event.target.name].splice(index, 1)
      this.setState({ filters })
    }
    const filteredProjects = this.props.projects.filter(project => {
      return this.projectContainsCause(project) && this.projectContainsTech(project)
    })
    this.setState({ filteredProjects })
  }
```